packages/data/src/useWalkthroughData.ts

```diff
@@ -122,15 +126,21 @@ export interface WalkthroughJSONType {
   };
 }
 
-const WalkthroughJSONData: Record<Walkthroughs, WalkthroughJSONType> = {
+export const WalkthroughJSONData: Record<Walkthroughs, WalkthroughJSONType> = {
   "9.9.9": testCase999,
 };
 
-export default function useWalkthroughData({ id }: UseWalkthroughDataProps) {
-  // check if passed id is a key in WalkthroughJSONData
-  if (!Object.prototype.hasOwnProperty.call(WalkthroughJSONData, id)) {
+export default function useWalkthroughData({
+  id,
+}: UseWalkthroughDataProps): WalkthroughJSONType {
+  if (!id) {
```

Comment:
Since id is a required prop by our types, do we need this?

Reply:
I think that's fine and the other comment about not needing the check as long as we are on the same page with the assumption that the walkthrough `json` is in fact correct and should bomb the application (in lower envs) if we somehow mess it up and pass incorrect data.